test/PackageVersionsTest/InstallerTest.php

```diff
@@ -477,18 +425,12 @@ private function __construct()
     /**
      * @throws OutOfBoundsException If a version cannot be located.
      *
-     * @psalm-param key-of<self::VERSIONS> $packageName
      * @psalm-pure
      */
     public static function getVersion(string $packageName) : string
     {
-        if (isset(self::VERSIONS[$packageName])) {
-            return self::VERSIONS[$packageName];
-        }
-
-        throw new OutOfBoundsException(
-            'Required package "' . $packageName . '" is not installed: check your ./vendor/composer/installed.json and/or ./composer.lock files'
-        );
+        return InstalledVersions::getPrettyVersion($packageName)
+            . '@' . InstalledVersions::getReference($packageName);
     }
```

Comment:
Too much copy/paste I see this all over the changelog! Can we abstract this?

Reply:
Yup: will mostly be about deleting tests later